templates/web/base/auth/sign_out.html

```diff
@@ -1,7 +1,10 @@
 [% INCLUDE 'header.html', title = loc('Sign out'), bodyclass = 'fullwidthpage' %]
 
-<h1>[% loc('You have been signed out') %]</h1>
+<div class="confirmation-header">
+  <h1>[% loc('You have been signed out') %]</h1>
+  <p>[% tprintf( loc('Please feel free to <a href="%s">sign in again</a>, or go back to the <a href="/">front page</a>.'), c.uri_for('/auth') ) %]</p>
+</div>
 
-<p>[% tprintf( loc('Please feel free to <a href="%s">sign in again</a>, or go back to the <a href="/">front page</a>.'), c.uri_for('/auth') ) %]</p>
+[% TRY %][% INCLUDE next_steps.html %][% CATCH file %][% END %]
```

Comment:
This is fine, however I guess we might want to encourage people to create a `next_steps.html` template and so it might make sense to have a basic example one in `base` perhaps? Even an empty one, perhaps. Then the TRY/CATCHes wouldn't be needed as it would always be present.